packages/server/graphql/mutations/createPoll.ts

```diff
@@ -0,0 +1,129 @@
+import {GraphQLNonNull} from 'graphql'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import CreatePollPayload from '../types/CreatePollPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+
+import {GQLContext} from '../graphql'
+import CreatePollInput from '../types/CreatePollInput'
+import MeetingMemberId from 'parabol-client/shared/gqlIds/MeetingMemberId'
+import insertPoll from '../../postgres/queries/insertPoll'
+import insertPollOptions from '../../postgres/queries/insertPollOptions'
+import segmentIo from '../../utils/segmentIo'
+
+const MAX_POLL_OPTIONS = 4
```

Comment:
why 4? In other places we use 12 to signify "a few" (1 for each month). maybe that's too much here, but i worry 4 is too little. @enriquesanchez thoughts?

Reply:
I used 4 as this is defined in [AC](https://github.com/ParabolInc/parabol/issues/5146). We can change that to whatever we think is best, thought!